### PR TITLE
sync llnl and ornl ci

### PR DESCRIPTION
**Pull request by andre-merzky**
attempt to sync CI

### CODE REVIEW

**Review by mtitov — APPROVED** [review]
LGTM!

File: .gitlab/llnl-ci.yml [hunk 1]
```diff
@@ -129,7 +130,7 @@ sdk-test:
       done
     - spack load exaworks py-pytest
     - export RADICAL_PILOT_DBURL=${MONGODB_CONNECTION_STRING}?tlsAllowInvalidCertificates=true
```
Comment (mtitov) [inline .gitlab/llnl-ci.yml:132]: just a comment regarding keeping `RADICAL_PILOT_DBURL` within GitLab's Settings - CI/CD - Variables: it doesn't let to keep string value with `?` character and having it masked, that's why for such value we have this extra exporting

**Comment by mturilli** [comment]
@andre-merzky , can we merge this?

**Comment by andre-merzky** [comment]
The Ascent batch runner we used on ORNL's gitlab is gone (for me?), need to iterate with support again... :-/

File: .gitlab/ornl-ci.yml [hunk 2]
```diff
@@ -1,93 +1,146 @@
...
-  - build ascent
+  - build
   - test
-  - clean
+  - cleanup
 
-Spack Setup:
+.final_steps:
+  script: &finalize
+    - chgrp -fR exaworks $SPACK_WORK_DIR/spack || true
```
Comment (mtitov) [inline .gitlab/ornl-ci.yml:12]: group should be `csc449` (project for exaworks)

File: .gitlab/ornl-ci.yml [hunk 3]
```diff
@@ -1,93 +1,146 @@
...
+  stage: compile
+  needs:
+    - job: spack-setup
+  parallel:
+    matrix:
+      - HOST: [ "ascent" ]
+        SPACK_ENV_NAME: [ "ascent" ]
+  tags:
+    - $HOST
+    - shell
```
Comment (mtitov) [inline .gitlab/ornl-ci.yml:36]: `shell` -> `nobatch`

File: .gitlab/ornl-ci.yml (hunk 2 above)
Comment (andre-merzky) [reply .gitlab/ornl-ci.yml]: Thanks, fixed.

File: .gitlab/ornl-ci.yml (hunk 3 above)
Comment (andre-merzky) [reply .gitlab/ornl-ci.yml]: Tanks, fixed.

**Pull request merged**